metadata-ingestion/src/datahub/ingestion/source/common/subtypes.py

```diff
@@ -17,6 +17,8 @@ class DatasetSubTypes(StrEnum):
     POWERBI_DATASET_TABLE = "PowerBI Dataset Table"
     QLIK_DATASET = "Qlik Dataset"
     BIGQUERY_TABLE_SNAPSHOT = "Bigquery Table Snapshot"
+    SHARDED_TABLE = "sharded table"
```

Comment:
should this be capitalized?

Reply:
Ideally yes. Kept as is in favour of earlier usage. Not sure, if UI handles this case change well. for example - filtering by subtype could show two entries `"sharded table"` and `"Sharded Table"`. Doesn't seem so harmful to make the change. Will change this.

Reply:
![image](https://github.com/user-attachments/assets/aec21acc-e398-4ba7-a08c-fadaf50c99e3)

Reply:
This should be temporary behavior as eventually all sharded tables would get same subtype.